src/dev/precommit_hook/casing_check_config.js

```diff
@@ -98,6 +98,9 @@ export const IGNORE_DIRECTORY_GLOBS = [
   ...KEBAB_CASE_DIRECTORY_GLOBS,
   'src/babel-*',
   'packages/*',
+  'packages/core/server/*',
+  'packages/core/common/*',
+  'packages/core/public/*',
```

Comment:
I believe you can just use `packages/core/{common,public,server}/*`

Reply:
Looking at the code, the globber used seems to be `minimatch`, and looking https://www.npmjs.com/package/minimatch, the syntax seems to be `packages/core/+(common,public,server)/*`

However looking at https://github.com/elastic/kibana/blob/456b25b81a0a0af64d0131776c0f7403963291c9/src/dev/precommit_hook/casing_check_config.js#L26, it seems the `{a|b}` is already used at some place.

Still,

```node
> minimatch("src/core/server", "src/core/{server|public}")
false
> minimatch("src/core/server", "src/core/+(server|public)")
true
```

So I wasn't sure which format to use, which is why I used multiple entries instead.

Probably @spalger can help here.

Reply:
Nevermind, it's very early in the morning. Using the correct `,` separator instead of `|` works...

```
> minimatch("src/core/server", "src/core/{server,public}")
true
```